Library/Homebrew/test/rubocops/blank_spec.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+require "rubocops/blank"
+
+describe RuboCop::Cop::Homebrew::Blank do
+  subject(:cop) { described_class.new }
+
+  shared_examples "offense" do |source, correction, message|
```

Comment:
Stupid question: would it be possible to use kwargs here to improve readability?

Reply:
Possibly! Though I should have added that the test cases are copy/pasta from upstream, where the test cases are appropriate, and I'm not especially inclined to drift from it:

https://github.com/rubocop/rubocop-rails/blob/master/spec/rubocop/cop/rails/blank_spec.rb#L4

(Source code made available under the MIT [license](https://github.com/rubocop/rubocop-rails/blob/master/LICENSE.txt), which may be appropriate to paste into the new files here, depending on one's view of "substantial portions".)